packages/teams-js/test/public/navigation.spec.ts

```diff
@@ -21,134 +22,173 @@ describe('MicrosoftTeams-Navigation', () => {
     }
   });
 
-  describe('returnFocus', () => {
-    it('should successfully returnFocus', async () => {
-      await utils.initializeWithContext('content');
+  describe('Testing navigation.returnFocus function', () => {
+    const allowedContexts = [FrameContexts.content];
 
-      returnFocus(true);
-
-      const returnFocusMessage = utils.findMessageByFunc('returnFocus');
-      expect(returnFocusMessage).not.toBeNull();
-      expect(returnFocusMessage.args.length).toBe(1);
-      expect(returnFocusMessage.args[0]).toBe(true);
-    });
-  });
-  describe('navigateCrossDomain', () => {
-    it('should not allow calls before initialization', () => {
-      expect(() => navigateCrossDomain('https://valid.origin.com')).toThrowError(
-        'The library has not yet been initialized',
-      );
+    it('navigation.returnFocus should not allow calls before initialization', () => {
+      expect(() => returnFocus(true)).toThrowError('The library has not yet been initialized');
```

Comment:
returnFocus just calls the pages.returnFocus function.  
```
export function returnFocus(navigateForward?: boolean): void {
  pages.returnFocus(navigateForward);
}
```
We should test pages.returnFocus in pages.spec file. I don't think we need to duplicate the tests here

Reply:
These are deprecated function and are internally replaced by `pages.returnFocus`. Here in this unit test case, we are checking even if we call deprecated function it should work.

Reply:
Why just checking that `pages.returnFocus` has been called (or not) does not work? I am just trying to understand.

Reply:
It will work but the scope is to see the expected output when depreciated function is used and is working accordingly.

Reply:
Check the latest commit I have added new test cases as you mentioned above.